### PR TITLE
Add documentation about cleanup properties

### PR DESCRIPTION
**Pull request by Martchus**
Also add  basics about job groups because
cleanup properties work on job-group-level

### CODE REVIEW

File: docs/GettingStarted.asciidoc [hunk 1]
```diff
@@ -201,6 +201,42 @@ of the openQA security model, refer to the
...
+  finished
+* keep results for: specifies how long results of a non-important job are retained
+  after it finished
+* keep important results for: how long results of an important job are retained after
+  it finished
+
+Note that deletion of job results includes deletion of logs and will cause the job to
+be completely removed from the database.
+
+Also note that jobs which do not belong to a job group are currently not affected by
```
Comment (foursixnine) [inline docs/GettingStarted.asciidoc]: This can can also use the NOTE: or TIP: macro

File: docs/GettingStarted.asciidoc [hunk 2]
```diff
@@ -201,6 +201,42 @@ of the openQA security model, refer to the
...
+* keep logs for: specifies how long logs of a non-important job are retained after
+  it finished
+* keep important logs for: how long logs of an important job are retained after it
+  finished
+* keep results for: specifies how long results of a non-important job are retained
+  after it finished
+* keep important results for: how long results of an important job are retained after
+  it finished
+
+Note that deletion of job results includes deletion of logs and will cause the job to
```
Comment (foursixnine) [inline docs/GettingStarted.asciidoc]: you could use the *WARNING:* macro, and change it to "The deletion..." or even "The cleanup of...".

File: docs/GettingStarted.asciidoc [hunk 3]
```diff
@@ -201,6 +201,49 @@ of the openQA security model, refer to the
...
+keep important results for:: How long results of an important job are retained after
+  it finished
+
+The defaults for those values are defined in
+https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Schema/JobGroupDefaults.pm[lib/OpenQA/Schema/JobGroupDefaults.pm].
+
+*NOTE* Deletion of job results includes deletion of logs and will cause the job to
+be completely removed from the database.
+
+Also note that jobs which do not belong to a job group are currently not affected by
```
Comment (aaannz) [inline docs/GettingStarted.asciidoc]: I would repeat the same format as in previous note:
`*NOTE*  jobs which do not ....`
instead of 'Also note'.

Comment (Martchus) [inline docs/GettingStarted.asciidoc]: Fixed

**Pull request merged**